### ISSUE TITLE
[Game Support]: CarX Street unable to load world

### ISSUE THREAD

**Issue by kartikdutt-vip** [PlayCover/PlayCover#1020]
### App Store link

https://apps.apple.com/us/app/carx-street/id1458863319

### Current issues

- [ ] Game fails to install
- [X] Game fails to open
- [ ] Game crashes randomly during gameplay
- [ ] Game does not work with keymapping
- [ ] Game experiences graphical issues or broken UI
- [X] Game fails to connect to servers or complete download

### More details

When I downloaded and installed the game it didn't asked me for storage permission but as I got to the stage where it needs to download assets for the game it simply says an unknown error occurred please try again I have tried every thing like changing the resolution and also tried using 3.0.0 build  of play cover but no luck and also  disabled and re enabled SIP and cleared both cache  and app data but still no luck can you please help me regarding my issue 
I will also attach a picture of the error screen 
<img width="1680" alt="Screenshot_2023-06-13_at_1 22 08_PM" src="https://github.com/PlayCover/PlayCover/assets/61133232/f350edc2-8ea0-4c32-9612-f0a0890e0fcb">

Thank You 

### Issue Language

- [X] Yes my issue is written in English

**Comment by alishtamimi** [comment]
Any updates guys here ?

**Comment by Heyamahah** [comment]
I have the same issue as well, the ipa was from decrypt.day

**Comment by sackboy19** [comment]
I'm having the same issue, however when I disabled SIP and AMFI, then enabled Jailbreak Bypass and Insert Introspection libraries in the PlayCover settings, I get an infinite loading screen with no error

https://github.com/PlayCover/PlayCover/assets/83356017/b6083ad0-e076-4d55-bf44-00d8fa9f4154

**Comment by Heyamahah** [comment]
> I'm having the same issue, however when I disabled SIP and AMFI, then enabled Jailbreak Bypass and Insert Introspection libraries in the PlayCover settings, I get an infinite loading screen with no error
> 
>  Screen.Recording.2023-12-15.at.7.22.35.AM.mov

Did the game go through successfully?

**Comment by alishtamimi** [comment]
No it doesn't, it just keep loading forever, I believe that the game itself is preventing its content to be downloaded or ran on macOS or Windows, I have tried to run this game over and over in every possible way in windows and macOS but with no luck at all.

**Comment by WwLEVRAMww** [comment]
Same Issue
![image](https://github.com/PlayCover/PlayCover/assets/100520080/c050be6e-687f-4bc4-bfb9-d6a48b02cf22)

**Comment by Sailor249** [comment]
I found a solution to the proplem if u still interested let me know

**Comment by kartikdutt-vip** [comment]
Yup I also got the solution just side loaded it using sideloadly and connected a ps controller (tested with iosgods version of the game not the official decrypted ipa)

**Comment by Sailor249** [comment]
> Yup I also got the solution just side loaded it using sideloadly and connected a ps controller (tested with iosgods version of the game not the official decrypted ipa)

i also use iosgods version but there is new update and i can only use offical decrypted ipa do you know how i could make it work with it?

**Comment by yogithesymbian** [comment]
any update?

**Comment by yogithesymbian** [comment]
<img width="1680" alt="Screenshot 2024-08-07 at 4 01 49 AM" src="https://github.com/user-attachments/assets/a29e6ff9-f3a9-4ba7-ade5-07625b0f34b5">

nighly build version  mbp m1 ( 8GB )

**Comment by yogithesymbian** [comment]
### when i tried :

<img width="215" alt="Screenshot 2024-08-07 at 4 08 04 AM" src="https://github.com/user-attachments/assets/1c128ac4-a176-4a9b-85e3-c63a392574ff">

### i got these errors :

<img width="1680" alt="Screenshot 2024-08-07 at 4 07 39 AM" src="https://github.com/user-attachments/assets/f05b3b9d-d261-48b5-9b1a-c38d44d382c9">

**Comment by kartikdutt-vip** [comment]
@yogithesymbian bro you have to download the cracked ipa from iosgods the official app still doesn't work the iosgods ipa comes with the assets pre downloaded and instead of using playcover side load the ipa on mac using sideloadly and connect a controller and you got a working game

**Comment by ProhorTaim** [comment]
> @yogithesymbian pre downloaded and instead of using playcover side load the ipa on mac using sideloadly and connect a controller and you got a working game

If I don't have a controller, how can I use keymapping?

**Comment by ProhorTaim** [comment]
And is there a performance difference between playcover and sideloadly?

**Comment by yogithesymbian** [comment]
> @yogithesymbian bro you have to download the cracked ipa from iosgods the official app still doesn't work the iosgods ipa comes with the assets pre downloaded and instead of using playcover side load the ipa on mac using sideloadly and connect a controller and you got a working game 

i dont understand.
using playcover side load
using sideloadly

what do u mean ? i only have macOS ( macbook ) and playcover only . 

1. device : macbook
2. file : ipa download
3. apps : playcover

**Comment by Royjumpstart** [comment]
> @yogithesymbian bro you have to download the cracked ipa from iosgods the official app still doesn't work the iosgods ipa comes with the assets pre downloaded and instead of using playcover side load the ipa on mac using sideloadly and connect a controller and you got a working game

iosgods only have the modded one

**Comment by Royjumpstart** [comment]
> > @yogithesymbian bro you have to download the cracked ipa from iosgods the official app still doesn't work the iosgods ipa comes with the assets pre downloaded and instead of using playcover side load the ipa on mac using sideloadly and connect a controller and you got a working game
> 
> i dont understand. using playcover side load using sideloadly
> 
> what do u mean ? i only have macOS ( macbook ) and playcover only .
> 
> 1. device : macbook
> 2. file : ipa download
> 3. apps : playcover

i've been waited for 6 months and it's still stuck! no answer from playcover dev. 
I tried using playcover recent nightly build version and recent normal version, but the output still the same.
